CHANGES.txt

```diff
@@ -1,3 +1,109 @@
+2015-08-26 version 3.0.0-beta-1 (C++/Java/Python/Ruby/Nano/Objective-C/C#)
+  About Beta
+  * This is the first beta release of protobuf v3.0.0. Not all languages
+    reaches beta stage. Except for those languages marked as beta, others
```

Comment:
Not all languages have reached beta stage.  Languages not marked as beta are still in alpha.